#this message will be used for IR ranger sensor and ultrasonic range sensor
RangeDefined[] ranges
